vasyl_varyshchuk/Week_2/task_1/func_in_threads.py

```diff
@@ -0,0 +1,97 @@
+""""
+This module returns random size of file name from list by processing elements in separate threads
+"""
+
+import random
+import string
+import Queue
+from threading import Thread
+
+
+number_of_threads = 5
+len_of_list = number_of_threads * 100
```

Comment:
Bad name of the variable.
It tells about nothing.

Reply:
Ok. I renamed this variable.